src/main/resources/mybatis/collections.xml

```diff
@@ -102,4 +102,48 @@
         ) as collection
     </select>
 
+
+    <select id="getStatisticalTimeSeriesJson" parameterType="String" resultType="String">
```

Comment:
parameterType="String" is misleading. A Map is being used.

Reply:
Oh, id did not notice that. Copy paste from other queries. I will look to see what that should be. Thanks.

Reply:
So I am going to make this "java.util.Map" -- is that correct? I bet so.